waltz-tools/src/test/java/com/wepay/waltz/tools/server/ServerCliTest.java

```diff
@@ -99,7 +102,8 @@ public void testAddAndRemovePreferredPartition() throws Exception {
         Properties cfgProperties = createProperties(helper.getZkConnectString(), helper.getZnodePath(), helper.getSslSetup());
         String configFilePath = IntegrationTestHelper.createYamlConfigFile(DIR_NAME, CONFIG_FILE_NAME, cfgProperties);
 
-        int partitionId = 3;
+        String partitionIdInterval = "3,6-7";
```

Comment:
may be `partitionIdRange` instead of `partitionIdInterval` so that it will be easy to understand.